R/pr.R

```diff
@@ -629,14 +632,10 @@ pr_clean <- function(number = NULL,
 # we're in DEFAULT branch of a fork. I wish everyone set up DEFAULT to track the
 # DEFAULT branch in the source repo, but this protects us against sub-optimal
 # setup.
-pr_pull_source_override <- function(tr = NULL, default_branch = NULL) {
-  # naive selection of target repo; calling function should analyse the config
-  tr <- tr %||% target_repo(github_get = FALSE, ask = FALSE)
```

Comment:
I definitely remember that the only reason I had `NULL` default here and did this was for development convenience.

Reply:
My thinking was that it's worth a little development inconvenience in order to ensure that we get efficient code.